en/ui-system/components/editor/canvas.md

```diff
@@ -0,0 +1,31 @@
+# Canvas Component Reference
+
+The **Canvas** component can obtain the actual resolution of the device screen at any time and zoom in / out of all the rendered elements in the scene. There can be multiple Canvas in the scene, and all UI elements must be placed under Canvas and its child nodes to be rendered.
```

Comment:
The **Canvas** component is in charge of rendering all UI components, all UI elements must be placed under a Canvas so that it can be rendered. The **Canvas** component can adapt the UI elements in designed resolution to the actual resolution of the device screen and can control zoom of all UI elements. There can be multiple Canvas in the scene, each of them represents a camera, the rendering results of each canvas and 3d cameras can be presented in a certain order controlled by the `priority`.

Reply:
这部分中文的描述可能是有很大问题的，@JoneLau 这里说的 Canvas 可以获取分辨率，缩放等并不是重要的部分，重要的是 Canvas 负责呈现 UI，并且所有 UI 都应该被放置在它的子节点树上，通过 canvas 的 camera 缩放的也不是 所有的渲染元素，而是相关的 UI 元素。